CHANGELOG.md

```diff
@@ -120,6 +120,7 @@ for the raw sql migrations.
 * Reduction in sync-time.
 * Added `--disable-cache` flag, which slightly reduces memory usage. Read more at `doc/configuration.md`.
 * Renamed `--no-epoch-table` flag to `disable-epoch`. Read more at `doc/configuration.md`.
+* Renamed table `instant_reward` to `reward_rest`.
```

Comment:
I think this was done later, since `instant_reward` was introduced in `13.2.0.1`

Reply:
The github UI is so confusing https://github.com/IntersectMBO/cardano-db-sync/commit/3b20b03ed79567c2a9d30b4b460d85e5f9bfc3b3

<img width="415" alt="image" src="https://github.com/user-attachments/assets/74eb079b-1c78-417f-b620-9dcfa46623d3">
But yeah looks like it was 3 months ago which would fit with `13.2.0.1`?